environment.yml

```diff
@@ -0,0 +1,66 @@
+name: cognoma-cancer-data
+dependencies:
```

Comment:
@clairemcleod, I attempted to fix the issue (see https://github.com/cognoma/cancer-data/pull/7/commits/8fb66e94fec1b1df48121f30b45c3a492082c0a3). Do you want to see whether the environment works for you on OS X now?